packages/strapi-plugin-users-permissions/admin/src/containers/AuthPage/index.js

```diff
@@ -143,67 +162,67 @@ export class AuthPage extends React.Component { // eslint-disable-line react/pre
   }
 
   renderInputs = () => {
-    const { match: { params: { authType } } } = this.props;
+    const { didCheckErrors, formErrors, match: { params: { authType } }, modifiedData, noErrorsDescription, onChangeInput, submitSuccess } = this.props;
     const inputs = get(form, ['form', authType]);
 
     return map(inputs, (input, key) => (
       <Input
         autoFocus={key === 0}
         customBootstrapClass={get(input, 'customBootstrapClass')}
-        didCheckErrors={this.props.didCheckErrors}
-        errors={get(this.props.formErrors, [findIndex(this.props.formErrors, ['name', input.name]), 'errors'])}
+        didCheckErrors={didCheckErrors}
+        errors={get(formErrors, [findIndex(formErrors, ['name', input.name]), 'errors'])}
         key={get(input, 'name')}
-        label={authType === 'forgot-password' && this.props.submitSuccess? { id: 'users-permissions.Auth.form.forgot-password.email.label.success' } : get(input, 'label')}
+        label={this.isAuthType('forgot-password') && submitSuccess? { id: 'users-permissions.Auth.form.forgot-password.email.label.success' } : get(input, 'label')}
```

Comment:
Isn't better move logic to is own sentence and promote the declarative intention of JSX?. Plus, we avoiding this long lines and making more readable.

Reply:
I agree